simulariumio/tests/filters/test_translate_filter_sphere_groups.py

```diff
@@ -0,0 +1,1213 @@
+                                -0.07831207665480905,
+                                0.1882706,
+                                -0.2239699592405966,
+                                -0.34936758585870953,
+                                0.133708233345191,
+                                0.1882706,
+                                0.08982416075940414,
+                                -0.30619373585870946,
+                                0.12200107334519095,
+                                0.1882706,
+                                0.3608457507594034,
+                                -0.3351345458587094,
+                                0.153759533345191,
+                                0.1882706,
+                                -0.39592128924059633,
+                                -0.12325413585870937,
+                                0.14518854334519093,
+                                0.1882706,
+                                -0.054375799240596656,
+                                -0.05610295585870917,
+                                0.14127757334519098,
+                                0.1882706,
+                                0.23853449075940336,
+                                -0.07756553585870929,
+                                0.14562730334519092,
+                                0.1882706,
+                                0.5361469007594035,
+                                -0.07129567585870955,
+                                0.08391924334519096,
+                                0.1882706,
+                                -0.5219094992405964,
+                                0.16691234414129053,
+                                0.11812568334519091,
+                                0.1882706,
+                                -0.22127055924059658,
+                                0.1524363041412906,
+                                0.15388547334519098,
+                                0.1882706,
+                                0.09807903075940327,
+                                0.20389651414129073,
+                                0.12309504334519095,
+                                0.1882706,
+                                0.3936103607594035,
+                                0.19296930414129054,
+                                0.15978946334519095,
+                                0.1882706,
+                                1000,
+                                10025,
+                                3,
+                                -0.58512353,
+                                -0.15074822,
+                                -0.7327957,
+                                0,
+                                0,
+                                0,
+                                1.0,
+                                4,
+                                0,
+                                0,
+                                0,
+                                0.1882706,
```

Comment:
does this data really need to be this large to demonstrate the test is working? It makes it impossible to intuitively see what this test is doing.

Reply:
I shortened this new test to only check the bundle data for 1 frame, since this filter only changes the positions and subpoints. In order to run filter() I need a converter, so still have to create a full trajectory as input to the test, but the expected data is now only the bundle data for 1 frame. 

Breaking up the other tests that I updated in this PR will be done in https://github.com/simularium/simulariumio/issues/98